packages/theme/src/cli/utilities/theme-environment/proxy.ts

```diff
@@ -29,7 +29,7 @@ export const VANITY_CDN_PREFIX = '/cdn/'
 export const EXTENSION_CDN_PREFIX = '/ext/cdn/'
 
 const CART_PATTERN = /^\/cart\//
-const ACCOUNT_PATTERN = /^\/account\/?$/
+const ACCOUNT_PATTERN = /^\/account(\/login\/multipass\/[^/]+)?\/?$/
```

Comment:
Would there be any downsides to modifying this pattern to be more flexible by accepting anything under `/account`?

Currently, we render a 401 if we don't provide a trailing `/` after `multipass`: 
`https://mammoth-matcha.myshopify.com/account/login/multipass`

This is what it looks like in prod:
<img width="961" alt="image" src="https://github.com/user-attachments/assets/94aeec76-9431-4aed-ac65-afb6b8d845f5">

cc: @karreiro